File: Serene3/Serene3.Web/Modules/com/payments/paymentsForm.cs
using Serenity.ComponentModel;
using System;

namespace Serene3.com.Forms;

[FormScript("com.payments")]
[BasedOnRow(typeof(paymentsRow), CheckNames = true)]
public class paymentsForm
{
    public int OrderId { get; set; }
    public string Provider { get; set; }
    public string ProviderPaymentId { get; set; }
    public decimal Amount { get; set; }
    public string Currency { get; set; }
    public string Status { get; set; }
    public string RawResponse { get; set; }
    public DateTime CreatedAt { get; set; }
    public DateTime UpdatedAt { get; set; }
}

File: Serene3/Serene3.Web/Modules/ServerTypes/com/paymentsForm.ts
import { DateEditor, DecimalEditor, initFormType, PrefixedContext, ServiceLookupEditor, StringEditor } from "@serenity-is/corelib";

export interface paymentsForm {
    OrderId: ServiceLookupEditor;
    Provider: StringEditor;
    ProviderPaymentId: StringEditor;
    Amount: DecimalEditor;
    Currency: StringEditor;
    Status: StringEditor;
    RawResponse: StringEditor;
    CreatedAt: DateEditor;
    UpdatedAt: DateEditor;
}

export class paymentsForm extends PrefixedContext {
    static readonly formKey = 'com.payments';
    private static init: boolean;

    constructor(prefix: string) {
        super(prefix);

        if (!paymentsForm.init) {
            paymentsForm.init = true;

            var w0 = ServiceLookupEditor;
            var w1 = StringEditor;
            var w2 = DecimalEditor;
            var w3 = DateEditor;

            initFormType(paymentsForm, [
                'OrderId', w0,
                'Provider', w1,
                'ProviderPaymentId', w1,
                'Amount', w2,
                'Currency', w1,
                'Status', w1,
                'RawResponse', w1,
                'CreatedAt', w3,
                'UpdatedAt', w3
            ]);
        }
    }
}